it/uk/gov/hmrc/agentrelationships/controllers/RelationshipsControllerISpec.scala

```diff
@@ -461,19 +458,18 @@ class RelationshipISpec extends UnitSpec
 
   "GET /agent/:arn/service/HMRC-MTD-VAT/client/VRN/:vrn" should {
 
-    val requestPath: String = s"/agent-client-relationships/agent/$arn/service/HMRC-MTD-VAT/client/VRN/$vrn"
+    val requestPath: String = s"/agent-client-relationships/agent/${arn.value}/service/HMRC-MTD-VAT/client/VRN/${vrn.value}"
 
     def doRequest = doAgentGetRequest(requestPath)
 
     //HAPPY PATH :-)
 
-    "return 200 when relationship exists in gg" in {
-      givenAgentCredentialsAreFoundFor(Arn(arn), "foo")
-      givenAgentCodeIsFoundFor("foo", "bar")
+    "return 200 when relationship exists in es" in {
+      givenPrincipalUser(arn, "foo")
+      givenGroupInfo("foo", "bar")
```

Comment:
`givenGroupInfo("foo", "bar")` stub already called in `givenAgentIsAllocatedAndAssignedToClient(_, "bar")`